Expensify/expensify-common
# Fix: multi level blockquote HTML to Markdown

This PR fixes the issue of inconsistency in Block Quote Usage Between the saved HTML and how it's rendered into markdown.

https://github.com/Expensify/expensify-common/assets/12425932/4c3afc10-ba13-4f38-8eb6-58801b52e6e3


### Fixed Issues
$ GH_LINK https://github.com/Expensify/App/issues/40571

# Tests
Unit tests added in the PR

# QA

1. bump expensify-common version in E/App
2. On Workspace description, use ">>> ABC"
3. save the new description and edit it again
4. verify that the displayed value to edit is the same ">>> ABC" and not "> ABC"
